tests/common/utils.rs

```diff
@@ -123,25 +123,24 @@ pub fn get_pubkey_from_input(vin: &VinData) -> Result<Option<PublicKey>, Error>
         }
     } else if is_p2sh(&vin.script_pub_key) {
         match (&vin.txinwitness.is_empty(), &vin.script_sig.is_empty()) {
-            (true, false) => {
+            (false, false) => {
                 let redeem_script = &vin.script_sig[1..];
```

Comment:
Maybe I'm misunderstanding something, but shouldn't this be `&vin.script_sig` without removing the first byte?

Reply:
Nevermind this, it's indeed correct I think